### PR TITLE
Issue33 api calls doc

### PR DESCRIPTION
**Pull request by stxphcodes**
Updated the API doc with more API calls and tables to describe user inputs

### CODE REVIEW

**Comment by seg1129** [comment]
Can you label each fields as required or optional.

File: ehb_service/apps/core/models/identities.py [hunk 1]
```diff
@@ -373,10 +373,11 @@ def responseFieldDict(self):
         return response
 
     def __unicode__(self):
-        return "{0}, {1} IN {2}".format(
+        return "{0}, {1} IN {2} ID {3}".format(
```
Comment (seg1129) [inline ehb_service/apps/core/models/identities.py]: It looks like 'ID {3}' was added by accident. please remove

File: ehb_service/apps/core/models/identities.py [hunk 2]
```diff
@@ -373,10 +373,11 @@ def responseFieldDict(self):
...
             self.subject.last_name,
             self.subject.first_name,
-            self.external_system.name
+            self.external_system.name,
+            self.record_id
```
Comment (seg1129) [inline ehb_service/apps/core/models/identities.py]: it looks like 'self.record_id' was added by accident, please remove.

File: docs/api.rst [hunk 3]
```diff
@@ -0,0 +1,1361 @@
...
+                "subject": 2
+            }
+        ],
+        "path": "Test Protocol",
+        "subject_id": "2",
+        "external_system_id": "2"
+    }
+]
+
+POST to obtain create an external Record
```
Comment (seg1129) [inline docs/api.rst]: why do you have 'obtain and create' here?

**Review by seg1129 — CHANGES_REQUESTED** [review]
there are just a few small changes that need to be made before I can commit.

File: docs/api.rst (hunk 3 above)
Comment (szh2425) [reply docs/api.rst]: 73ede78580a4bc68c23235a769e476bb1b6fee36

File: ehb_service/apps/core/models/identities.py (hunk 1 above)
Comment (szh2425) [reply ehb_service/apps/core/models/identities.py]: ec3aa9077077640eefcc2280e0c19c1d414eaaff

File: ehb_service/apps/core/models/identities.py (hunk 2 above)
Comment (szh2425) [reply ehb_service/apps/core/models/identities.py]: ec3aa9077077640eefcc2280e0c19c1d414eaaff

**Comment by seg1129** [comment]
@szh2425 please rebase this branch, then I can approve this PR.

**Review by seg1129 — APPROVED** [review]

**Review by seg1129 — CHANGES_REQUESTED** [review]
please remove changes made in ehb_service/apps/core/models/identities.py file.

**Review by seg1129 — APPROVED** [review]

**Pull request merged**